lib/static_sync/processor.rb

```diff
@@ -46,6 +46,8 @@ def handle_conflict(file)
         if file.cached?
           raise ConflictError, "modifications to existing cached files are not allowed."
```

Comment:
But we are using `'conflict_mode' => 'fail_if_cached'` in the code instead of `@config.fail_on_conflict_if_cached?`, is it the same thing? It looks confusing.